src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("16.0.0.0")]
-[assembly: AssemblyFileVersion("16.0.0.0")]
+[assembly: AssemblyVersion("16.1.0.0")]
```

Comment:
@EvgenyAgafonchikov please do not update AssemblyVersion for minor/build version changes. AssemblyVersion only changes with major version bumps.